autoconfigure/storage-applicationinsights/src/main/resources/zipkin-server-applicationinsights.yml

```diff
@@ -0,0 +1,7 @@
+# When enabled, this allows shorter env properties (ex -Dspring.profiles.active=applicationinsights)
+zipkin:
+  storage:
+    applicationinsights:
+      instrumentationKey: ${AI_INSTRUMENTATION_KEY:}
```

Comment:
environment variable for instrumentation key that we typically use is **`APPINSIGHTS_INSTRUMENTATIONKEY` https://github.com/Microsoft/ApplicationInsights-dotnet/blob/151509eac5e23fcbd09f3a8989a35d1099703957/src/Core/Managed/Shared/Extensibility/Implementation/TelemetryConfigurationFactory.cs#L22** 

It is not ideal as it uses a wrong name `appinishgts`, but it is what we use everywhere